README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+
+# HW01
+## About me
+
+Hello, my name is **Eunsil Choi**, a MA student in Committee on International Relations.  
```

Comment:
Be careful with **unnecessary spacing**. This will become more important in future assignments when we start to pay closer attention to **coding style**

Reply:
Hello, Havier. 
Do you mean the spacing between "##About me" and the fist line of the introduction?

Reply:
Javier, 
Do you mean unnecessary spacing between ##About me and the fist line of the introduction?